LarkatorGUI/SettingsWindow.xaml

```diff
@@ -109,7 +110,79 @@
                                                IsChecked="{Binding Source={StaticResource Model}, Path=Settings.TeleportGhost, Mode=TwoWay}" HorizontalAlignment="Right"
                                                Margin="7,4"/>
             </Border>
+            <Border Grid.Row="9" Grid.Column="0">
+                <TextBlock VerticalAlignment="Center" Margin="0,0,4,0">Hide untameable dinosaurs</TextBlock>
+            </Border>
+            <Border Grid.Row="9" Grid.Column="1">
+                <CheckBox ToolTip="Hide dinosaurs that cannot be tamed"
+                                               IsChecked="{Binding Source={StaticResource Model}, Path=Settings.hideUntameable, Mode=TwoWay}" HorizontalAlignment="Right"
+                                               Margin="7,4"/>
+            </Border>
         </Grid>
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="2.5*"/>
+                <ColumnDefinition Width="1*"/>
+                <ColumnDefinition Width="1*" />
+                <ColumnDefinition Width="1*" />
+                <ColumnDefinition Width="1*" />
+                <ColumnDefinition Width="1*" />
+                <ColumnDefinition Width="1*" />
+                <ColumnDefinition Width="1*" />
+                <ColumnDefinition Width="1*" />
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition />
+                <RowDefinition />
+            </Grid.RowDefinitions>
+            <Border Grid.Row="0" Grid.Column="2">
+                <TextBlock VerticalAlignment="Center" HorizontalAlignment="Right" Margin="7,4">Hp</TextBlock>
+            </Border>
+            <Border Grid.Row="0" Grid.Column="3">
+                <TextBlock VerticalAlignment="Center" HorizontalAlignment="Right" Margin="7,4">St</TextBlock>
+            </Border>
+            <Border Grid.Row="0" Grid.Column="4">
+                <TextBlock VerticalAlignment="Center" HorizontalAlignment="Right" Margin="7,4">We</TextBlock>
+            </Border>
+            <Border Grid.Row="0" Grid.Column="5">
+                <TextBlock VerticalAlignment="Center" HorizontalAlignment="Right" Margin="7,4">Me</TextBlock>
+            </Border>
+            <Border Grid.Row="0" Grid.Column="6">
+                <TextBlock VerticalAlignment="Center" HorizontalAlignment="Right" Margin="7,4">Sp</TextBlock>
+            </Border>
+            <Border Grid.Row="0" Grid.Column="7">
+                <TextBlock VerticalAlignment="Center" HorizontalAlignment="Right" Margin="7,4">Fd</TextBlock>
+            </Border>
+            <Border Grid.Row="0" Grid.Column="8">
+                <TextBlock VerticalAlignment="Center" HorizontalAlignment="Right" Margin="7,4">Ox</TextBlock>
+            </Border>
+
+            <Border Grid.Row="1" Grid.Column="0" Grid.ColumnSpan="2">
+                <TextBlock VerticalAlignment="Center" Margin="0,0,4,0">Display Stats:</TextBlock>
```

Comment:
Could you match the text style of the window and just make it "Display stats" please.

Reply:
Display text fixed in commit 7410d47ac54b244e61ddce82c10eb75f8d8d9281